### PR TITLE
config(raster): Store disable layers as standalone json files in the basemaps config. BM-808

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: config/imagery/tauranga_urban_2017_0-1m.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "type": "raster",
+  "id": "ts_tauranga_urban_2017_0-1m",
```
Comment (blacha) [inline config/imagery/tauranga_urban_2017_0-1m.json:3]: I cannot view this record in basemaps?

```
https://basemaps.linz.govt.nz/?i=tauranga-urban-2017-0.1m # Current
https://basemaps.linz.govt.nz/?i=tauranga_urban_2017_0-1m # basemaps-config
```

Comment (blacha) [reply config/imagery/tauranga_urban_2017_0-1m.json:3]: I wonder which is in actually in use?

Comment (Wentao-Kuang) [reply config/imagery/tauranga_urban_2017_0-1m.json:3]: We want to format all the names from `tauranga_urban_2017_0-1m` to `tauranga-urban-2017-0.1m`

**Comment by Wentao-Kuang** [comment]
Close out of date PR. New changes in https://github.com/linz/basemaps-config/pull/766

**Pull request closed without merging**